v3/pkg/application/dialogs_linux.go

```diff
@@ -28,13 +30,9 @@ func (m *linuxDialog) show() {
 		parent, _ = window.(*WebviewWindow).NativeWindowHandle()
 	}
 
-	response := runQuestionDialog(pointer(parent), m.dialog)
-	if response >= 0 && response < len(m.dialog.Buttons) {
-		button := m.dialog.Buttons[response]
-		if button.Callback != nil {
-			go button.Callback()
-		}
-	}
+	InvokeAsync(func() {
+		runQuestionDialog(pointer(parent), m.dialog)
+	})
```

Comment:
Did you mean to completely remove the `if statement`?

Reply:
Accidentally got removed should be fixed